spec/support/fixtures/test/main.swift

```diff
@@ -1,7 +1,7 @@
 protocol RedundantProtocol {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/529aa7e414601b50d75108ce9802e8359e274382/spec/support/fixtures/test/main.swift#L1"></span>Protocol 'RedundantProtocol' is redundant as it's never used as an existential type</td>
    </tr>
  </tbody>
</table>